src/migration.rs

```diff
@@ -21,29 +21,51 @@ use crate::{options::Options, db::Db, Error, Result, column::ColId};
 
 const COMMIT_SIZE: usize = 10240;
 
-pub fn migrate(from: &Path, mut to: Options) -> Result<()> {
-	let mut path: std::path::PathBuf = from.into();
-	path.push("metadata");
-	let source_meta = Options::load_metadata(&path)?
+pub fn migrate(from: &Path, mut to: Options, keep_dest: bool, force_migrate: &Vec<u8>) -> Result<()> {
```

Comment:
What is the `keep_dest` option for?

Reply:
I will rename it to overwrite, it changes a bit the semantic of migration.